src/Thermal/RadiativeFB.cpp

```diff
@@ -313,7 +277,10 @@ FLOAT SinkHeating<ndim>::AmbientTemp
 
     FLOAT dist      = Distance(part.r, sink.star->r, ndim);
     FLOAT sink_m    = sink.star->m;
-    FLOAT sink_dmdt = curr_mdot[i];
+    // TODO: This is accretion rate onto the sink, not the object at it's center. It may be
```

Comment:
Surely these are the same, unless it's a binary? But we don't handle the binary in any special way in any event.

Reply:
I'm making the point of distinguishing between the accretion rate onto the sink and the accretion rate onto the sink.star object. Maybe they are the same and I'm just overthinking it.